### PR TITLE
PowerDownBotModule up-to-date

### PR DESCRIPTION
**Pull request by dnqbob**
Some typo fixes and namespace fixes, make "OrderName" can be setting by users.

### CODE REVIEW

File: OpenRA.Mods.AS/Traits/BotModules/PowerDownBotASModule.cs [hunk 1]
```diff
@@ -52,33 +58,29 @@ public PowerDownBotASModule(Actor self, PowerDownBotASModuleInfo info)
 			world = self.World;
 			player = self.Owner;
 			toggledBuildings = new List<BuildingPowerWrapper>();
-			isToggledBuildingsValid = a => a.Owner == self.Owner && !a.IsDead && a.IsInWorld && GetTogglePowerChanging(a) < 0;
+			isToggledBuildingsValid = a => a.Owner == self.Owner && !a.IsDead && a.IsInWorld;
 		}
 
 		protected override void Created(Actor self)
 		{
-			// Special case handling is required for the Player actor.
```
Comment (GraionDilach) [inline OpenRA.Mods.AS/Traits/BotModules/PowerDownBotASModule.cs:60]: Why are you changing this? I'm not that confident in changing this one.

**Comment by GraionDilach** [comment]
The rest I'm tentatively okay with - but I don't remember any upstream change which made the Created special-casing redundant. Either pinpoint the relevant upstream PR fixing this circular dependency please or revert that one change for consistency.

File: OpenRA.Mods.AS/Traits/BotModules/PowerDownBotASModule.cs (hunk 1 above)
Comment (dnqbob) [reply OpenRA.Mods.AS/Traits/BotModules/PowerDownBotASModule.cs:60]: https://github.com/OpenRA/OpenRA/commit/aeab9a8116f51e7f467f37904658f7c22bc57276

So I changed that.

Comment (GraionDilach) [reply OpenRA.Mods.AS/Traits/BotModules/PowerDownBotASModule.cs:60]: Ah, https://github.com/OpenRA/OpenRA/pull/18182. Okay, that one I overlooked.

**Review by GraionDilach — APPROVED** [review]

**Comment by Mailaender** [comment]
I think `OrderName` should not be defined by modders as it is internal API. Otherwise this works fantastically. Thanks. https://github.com/OpenHV/OpenHV/commit/9c47794a52ed93bad6e9d20d79a77a4ba458d1f7

**Pull request merged**